RELEASE.md

```diff
@@ -1,3 +1,54 @@
+# Release 1.14.0
+
+## Features
+- New hyperparameters dashboard: see [tutorial and demo][hparams-docs] and
+  [summary APIs][hparams-apis]
+- New dashboard for visualizing meshes and point clouds: see
+  [README][mesh-readme]
+- Graph dashboard now shows Keras conceptual graph: see [tutorial and
+  demo][conceptual-graph-docs]
+- Embedding projector now supports the [UMAP dimensionality reduction
+  algorithm][umap] ([learn more about UMAP here][umap-tutorial]) (#1901) -
+  thanks @cannoneyed
+- [TensorBoard notebook support][notebook-docs] is stabilized: in a Colab or
+  Jupyter notebook, run `%load_ext tensorboard` followed by `%tensorboard
+  --logdir ./path/to/logs`
+- Profile dashboard improvements:
+  - New pod viewer tool to analyze TPU performance (#2111)
+  - Now allows capturing profiles from TensorBoard (#1894)
+- What-If Tool improvements:
+  - Now available as a notebook widget for Jupyter and Colab: see
+    [demo][witwidget-demo]
+  - Now shows PR curves and F1 score (#2264)
+  - Now supports Cloud AI Platform, including XGBoost models (#2194)
+  - Now shows feature-level attributions for individual predictions, as
+    applicable (#2252)
+- Image dashboard now allows scrolling for large images (#2164) - thanks @lr1d
+- Scalar chart smoothing now caps at 0.999 for convenience (#1974) - thanks
+  @flostim
+- Scalar chart scroll-to-zoom behavior now requires holding `Alt` (PR #2221)
```

Comment:
Bug fixes have “PR” (because the issue is the primary descriptor). I
originally had this listed as a bug but changed it to a feature and
forgot to change the wording. Will patch.